test/puppetlabs/trapperkeeper/services_test.clj

```diff
@@ -62,6 +64,67 @@
           app (bootstrap-services-with-empty-config [service1])]
       (is (not (nil? app))))))
 
+(deftest test-restart-file-correct-count
+  (testing "create a restart file and check that it correctly increments when a service starts"
+    (let [service1 (service Service1
+                            []
+                            (service1-fn [this] "hi"))
+          temp-file (kitchensink/temp-file-name "counter")]
```

Comment:
Just out of curiosity - did you find a reason to prefer `temp-file-name` here instead of just `temp-file`?  Was it maybe that `temp-file` would actually create the file and you didn't want it to do that?